TheMET/TheMET/CatalogSectionCellData/CatalogSectionCellData.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  CatalogSectionCellData.swift
+//  TheMET
+//
+//  Created by Анна Ситникова on 21/12/2023.
+//
+
+import Foundation
+import UIKit
+
+struct CatalogSectionCellData <T> {
```

Comment:
Now, when you have this generic class, you don't need `CatalogCellData`

Reply:
Done